pkgs/tools/graphics/perceptualdiff/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, freeimage }:
+
+stdenv.mkDerivation rec {
+  pname = "perceptualdiff";
+  name = "${pname}-${version}";
+  version = "1.2";
```

Comment:
v1.2 is from 2014 and the latest v2.1 from 2016 already. That's quite a version gap..

Reply:
Yes, as explained in the description I plan to update it, but this matches what is currently used on Ubuntu, even disco:  https://packages.ubuntu.com/source/disco/perceptualdiff

Reply:
#51249

Reply:
But nixpkgs does not need to match Ubuntu.
I'd vote to init the package at v2.1 directly and save another PR.